masterlist.yaml

```diff
@@ -10902,6 +10901,7 @@ plugins:
         name: 'Beyond Skyrim - Bruma SE'
   - name: 'BSAssets.esm'
     url: [ 'https://bethesda.net/en/mods/skyrim/mod-detail/4027835' ]
+    after: [ 'MajesticMountains_Landscape.esm' ]
```

Comment:
The after might not be required, as default sorting should plce it after.
It would have to be on `BSHeartland.esm` as well.